101-app-function/azuredeploy.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "webAppName": {
+            "type": "string",
```

Comment:
add a defaultValue as in other samples...

Reply:
Done